hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapreduce/AccumuloInputFormat.java

```diff
@@ -43,7 +43,7 @@
  * <pre>
  * AccumuloInputFormat.configure().clientProperties(props).table(name) // required
  *     .auths(auths).addIterator(iter1).ranges(ranges).fetchColumns(columns).executionHints(hints)
- *     .samplerConfiguration(sampleConf).disableAutoAdjustRanges() // enabled by default
+ *     .samplerConfiguration(sampleConf).autoAdjustRanges() // enabled by default
```

Comment:
Need to update javadoc to use boolean

Reply:
Thanks fixed in 338bb74

Reply:
It still needs to be fixed in AccumuloInputFormat & AccumuloRowInputFormat